test/config/integration/server.json

```diff
@@ -320,24 +320,6 @@
   {
     "address": "tcp://{{ ip_loopback_address }}:0",
     "filters": [
-      { "type": "read", "name":
```

Comment:
Is this config (and tests) removed because it is duplicated?

Reply:
Oops - updated PR description.  I also removed the duplicate TCP proxy tests (and config) because I noticed them.  Happy to split them out but they're at least called out in the PR description now!